lib/mix/lib/releases/models/app.ex

```diff
@@ -22,14 +22,7 @@ defmodule Mix.Releases.App do
   @valid_start_types [:permanent, :temporary, :transient, :load, :none]
 
   @doc """
-  Create a new Application struct from an application name
-  """
-  @spec new(atom) :: nil | __MODULE__.t() | {:error, String.t()}
```

Comment:
Can you add this function back? It's part of the public API and isn't relevant to the fix for the issue this PR targets